packit_service/service/models.py

```diff
@@ -141,12 +143,14 @@ def create(
         chroots: List[str],
         identifier: int = None,
         save: bool = True,
+        trigger_time: str = None,
```

Comment:
this is not very descriptive, how about we added three fields instead?
* build_submitted_time
* build_start_time
* build_finished_time